src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -475,6 +472,11 @@ def _delete_manifest_confirmation(login_server,
     if filter_by_manifest:
         images = ", ".join(["'{}:{}'".format(repository, str(x)) for x in filter_by_manifest])
         message += " and all the following images: {}".format(images)
+
+    if yes:
+        print(message)
```

Comment:
You cannot use a raw print statement. You need to use logger warning or redirect this to stderr instead of stdout.

Reply:
done